### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by elghanbibadr** [vercel/next.js#59223] [Help]
### Summary

meone helps fix this error I did a lot of searching but found nothing I'll really appreciate it if someone helps I installed a next js app and I removed all the content from the page component only kept this:  after I ran the script npm run I got this and thanks👏🥰
![nexterro2](https://github.com/vercel/next.js/assets/73561635/4867658a-89e3-469f-bd7c-c6226486df4c)
![nexterror](https://github.com/vercel/next.js/assets/73561635/97a4af25-1b82-4969-a9cc-3c07cec98b62)

**Comment by icyJoseph** [comment]
Did you try this in incognito mode? You should share more of the logs, usually the first log is the one that has the meaningful information.

**Comment by TrungNV31197** [comment]
Try this. I ran this command to update the NextJS version. It helped me resolve this error after one day of debugging.

`npm i next@rc react@rc react-dom@rc eslint-config-next@rc --force`

**Comment by elghanbibadr** [comment]
thank you so much

**Comment by mandorakannu** [comment]
Kindly share your `layout.js` & `next.config.js` file
